lib/symlinks.js

```diff
@@ -1,19 +1,27 @@
 const gutil = require('gulp-util');
+const minimist = require('minimist');
 const fs = require('fs');
 const path = require('path');
 const paths = require('./paths');
 
 const suffix = '.original';
+const options = minimist(process.argv.slice(2), {
+  string: 'symlinks',
+});
+
+const packages = options.symlinks.split(',');
 
 /**
  * Replaces vendor package directories with symlinks that point to active
  * package working directories instead of installed composer packages
  */
 function setup(err, progress, complete) {
   if (fs.existsSync(paths.vendors)) {
-    fs.readdirSync(paths.vendors).forEach((packageName) => {
+    packages.forEach((packageName) => {
       const packageLinkPath = path.join(paths.vendors, packageName);
-      if (fs.lstatSync(packageLinkPath).isDirectory()) {
+
+      if (fs.existsSync(packageLinkPath) &&
```

Comment:
should we throw an error if the package doesn't exist? or output a message to the command line?

Reply:
a warning message seems appropriate

Reply:
I now show a message